scaleway/resource_compute_instance_server.go

```diff
@@ -257,26 +269,24 @@ func resourceScalewayComputeInstanceServerRead(d *schema.ResourceData, m interfa
 		}
 		return err
 	}
-	switch response.Server.State {
-	case instance.ServerStateStopped:
-		d.Set("state", ServerStateStopped)
-	case instance.ServerStateStoppedInPlace:
-		d.Set("state", ServerStateStandby)
-	case instance.ServerStateRunning:
-		d.Set("state", ServerStateStarted)
-	case instance.ServerStateLocked:
-		return fmt.Errorf("server is locked, please contact Scaleway support: https://console.scaleway.com/support/tickets")
-	default:
-		return fmt.Errorf("server is in an invalid state, someone else might be executing action at the same time")
+	state, err := expandServerState(response.Server.State)
```

Comment:
by convention going from API format to state format is more a flattened than an expanded